### PR TITLE
Transition Hello World in Google Apps Script

### PR DESCRIPTION
**Pull request by JohnnyFang**
Congrats on taking the first step to contributing to the [Sample Programs in Every Language][1] repository!

For simplicity, please make sure that your pull request includes one and only one sample program and language.

## Please Complete the Following

- [ ] I would **NOT** like credit for my contribution in the article (credit will be included otherwise)
- [ ] I plan to write the article for this code snippet
- [ ] I named the pull request using `<Sample Program> in <Language>` format
- [ ] I created/updated the language README
  - [ ] I added the sample program name to the README
  - [ ] I added fun facts (i.e. debut, developer, typing, etc.)
  - [ ] I added reference link(s) to the README
  - [ ] I added solution citations when necessary (see [plagiarism][2])
- [ ] I fixed #your-issue-number-here

## Notes

Feel free to put whatever you want here.

[1]: https://therenegadecoder.com/code/sample-programs-in-every-language/
[2]: https://github.com/TheRenegadeCoder/sample-programs/blob/master/CONTRIBUTING.md#plagiarism

### CODE REVIEW

File: docs/languages/_posts/2018-11-10-google-apps-script.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+---
+title: The Google Apps Script Programming Language
+layout: default
+last-modified: 2018-11-10
+featured-image:
+tags: [google-apps-script]
+authors:
+  - jeremy-grifski
```
[the remark this answers is not in the document]
Comment (JohnnyFang) [reply docs/languages/_posts/2018-11-10-google-apps-script.md]: yes, you are correct. I just checked the authors.yml
I'll make the corrections

File: docs/projects/hello-world/_posts/2018-11-10-google-apps-script.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+Inside the code block, we have our typical print call. In this case, we leverage the Logger to do our printing. Then we pass our “Hello, World!” string to the log function, and call it a day.
+
+## How to Run the Solution
+
+If we want to run Hello World in Google Apps Script, we actually have to write our scripts using the Apps Script tool. From there, Google has some nice documentation for running them.
+
+Alternatively, we can write scripts locally and upload them to Google Drive. At that point, we can connect the Google Apps Script tool to run our scripts. The link above has directions for that option as well.
+
+If you know of other ways to run Google Apps Script code, let me know in the comments.
```
[the remark this answers is not in the document]
Comment (JohnnyFang) [reply docs/projects/hello-world/_posts/2018-11-10-google-apps-script.md]: yes, I did miss the reference. my bad!

File: docs/projects/hello-world/_posts/2018-11-10-google-apps-script.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+---
+
+#### References
+
+[^1]: J. Grifski, “Hello World in Google Apps Script,” The Renegade Coder, 22-May-2018. [Online]. Available: <https://therenegadecoder.com/code/hello-world-in-google-apps-script/>. [Accessed: 10-Nov-2018].
+
+[1]: https://therenegadecoder.com/code/hello-world-in-javascript/
+[2]: https://www.google.com/script/start/
```
[the remark this answers is not in the document]
Comment (JohnnyFang) [reply docs/projects/hello-world/_posts/2018-11-10-google-apps-script.md]: I added the link reference for these two.

File: docs/projects/hello-world/_posts/2018-11-10-google-apps-script.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
+---
+title: Hello World in Google Apps Script
+layout: default
+last-modified: 2018-11-10
+featured-image: hello-world-in-google-apps-script-featured-image.JPEG
+tags: [google-apps-script, hello-world]
+authors:
+  - jeremy-grifski
```
Comment (jrg94) [inline docs/projects/hello-world/_posts/2018-11-10-google-apps-script.md]: Missing an update here as well!

File: docs/projects/hello-world/_posts/2018-11-10-google-apps-script.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (jrg94) [reply docs/projects/hello-world/_posts/2018-11-10-google-apps-script.md]: I think these are still missing! Otherwise, it's good.

**Comment by jrg94** [comment]
I fixed up those footnotes! Merging now.

**Pull request merged**